### PR TITLE
DEXW-2781: Add sponsorship confirmation

### PR DESCRIPTION
**Pull request by finico**

### CODE REVIEW

File: src/iframe-entry/pages/SignSponsorship/SignSponsorshipComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+    sponsorAsset,
+    sponsorCharge,
+    isSponsorshipEnable,
+    onReject,
+    onConfirm,
+}) => (
+    <Confirmation
+        address={userAddress}
+        name={userName}
+        balance={`${getPrintableNumber(userBalance, WAVES.decimals)} Waves`}
```
Comment (mrpq) [inline src/iframe-entry/pages/SignSponsorship/SignSponsorshipComponent.tsx]: думаю можно унести в контейнер

File: src/iframe-entry/pages/SignSponsorship/SignSponsorshipComponent.tsx [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+                        icon={iconSponsorshipDisableTransaction}
+                        size={40}
+                        color="warning.$500"
+                    />
+                </Flex>
+            )}
+
+            <Flex ml="$20" alignItems="center">
+                <Text fontSize="26px" lineHeight="32px" color="standard.$0">
+                    Sign Enable Sponsorship TX
```
Comment (mrpq) [inline src/iframe-entry/pages/SignSponsorship/SignSponsorshipComponent.tsx]: кажется не хватает ифчика для disable

File: src/iframe-entry/pages/SignSponsorship/SignSponsorshipComponent.tsx (hunk 1 above)
Comment (finico) [reply src/iframe-entry/pages/SignSponsorship/SignSponsorshipComponent.tsx]: it's at this place in every component

File: src/iframe-entry/pages/SignSponsorship/SignSponsorshipComponent.tsx (hunk 2 above)
Comment (finico) [reply src/iframe-entry/pages/SignSponsorship/SignSponsorshipComponent.tsx]: oops

Comment (finico) [reply src/iframe-entry/pages/SignSponsorship/SignSponsorshipComponent.tsx]: fixed

File: src/iframe-entry/pages/SignSponsorship/SignSponsorshipComponent.tsx (hunk 1 above)
Comment (mrpq) [reply src/iframe-entry/pages/SignSponsorship/SignSponsorshipComponent.tsx]: Except from [there](https://github.com/waves-exchange/provider-web/commit/992cd43fbb4c98b1a14ccf8f169d8fcbc14d366a#diff-6bcebb7ad429542ca6019d74974728dcR68) ;)

Comment (finico) [reply src/iframe-entry/pages/SignSponsorship/SignSponsorshipComponent.tsx]: it's better to replace them by one shot sometime

**Pull request merged**